### PR TITLE
New content

### PR DESCRIPTION
**Pull request by Cleop**
New content specific to the operation of the London campus.

* Cleaning rota
* Environment
* Key people
* Meetup rota
* FAC10 project teams

Related #6

### CODE REVIEW

File: project_teams.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Project Teams
+
+In the first 8 weeks of the course you will complete weekly team projects. The schedule for these teams is as follows:
+
+| Week 1-3                     |  Week 4-6                     | Week 7-8                     |
+|----------------------------- |-------------------------------|------------------------------|
+|Yvonne, Jessica, Akin, Oliver |Yvonne, Lucy, Samatar, Finn    |Yvonne, Martha, Zooey, Antonio|
+|Lucy, Joey, Piotr, Zooey      |Jessica, Joey, Alexis, Antonio |Jessica, Alice, Piotr, Finn   |  
+|Martha, Alice, Samatar, Alexis|Akin, Piotr, Martha, Maja      |Lucy, Maja, Oliver, Alexis    |
```
Comment (bradreeder) [inline project_teams.md]: I think you're missing a column as the project teams here only include 12 people?

**Comment by bradreeder** [comment]
@Cleop I've added the calendar to the readme.md here and a file listing the mentors and their assigned weeks in #12 . I think we've got everything. 👍

**Comment by bradreeder** [comment]
@Cleop @jsms90 @njsfield I'm unsure whether we should let the students know the project teams for weeks 4-6 and 7-8. I'm happy to go with your discretion, but I think it reduces our flexibility.

**Comment by sohilpandya** [comment]
agree with @bradreeder in terms of teams for the future weeks.

**Comment by jsms90** [comment]
@bradreeder let's notify week 4 & week 7's mentors that they should update the file on day 1 of their week?

Although any curious student can always look in the history of the file, or this issue ;)

**Pull request merged**